src/components/sbb-menu-action/sbb-menu-action.tsx

```diff
@@ -12,72 +12,52 @@ let nextId = 0;
 
 /**
  * @slot unnamed - Use this slot to provide the menu action label.
- * @slot icon - Use this slot to provide an SVG icon. If `icon` is set, a sbb-icon will be used.
+ * @slot icon - Use this slot to provide an icon. If `icon` is set, an sbb-icon will be used.
  */
 @Component({
   shadow: true,
   styleUrl: 'sbb-menu-action.scss',
   tag: 'sbb-menu-action',
 })
 export class SbbMenuAction implements LinkButtonProperties {
-  /**
-   * This id will be forwarded to the relevant inner element.
-   */
-  @Prop() public menuActionId = `sbb-menu-action-${++nextId}`;
+  /** This id will be forwarded to the relevant inner element. */
+  @Prop() public idValue = `sbb-menu-action-${++nextId}`;
```

Comment:
Has the specification for the id name changed? In the Coding Standards it is still written that "The name of the property should be `inputId` for custom form elements and `{componentName}Id` for everything else."